src/consensus/aft/test/main.cpp

```diff
@@ -21,7 +21,7 @@ using StoreSig = aft::LoggingStubStoreSig;
 using Adaptor = aft::Adaptor<Store>;
 
 threading::ThreadMessaging threading::ThreadMessaging::thread_messaging;
-std::atomic<uint16_t> threading::ThreadMessaging::thread_count = 0;
+std::atomic<uint16_t> threading::ThreadMessaging::thread_count = 1;
```

Comment:
Tangential to this PR: Its confusing to have a possible value here (`0`) that most of the systems won't handle, and then have a load of `thread_count > 1` checks. I wonder if we can instead phrase this as `worker_thread_count`, and implicitly there's _always_ a main thread with 0-or-more worker threads.